src/util.ml

```diff
@@ -41,8 +41,14 @@ value search_in_lang_path = search_in_path Secure.lang_path;
 
 value start_with_vowel s =
   if String.length s > 0 then
-    match Char.lowercase s.[0] with
-    [ 'a' | 'e' | 'i' | 'o' | 'u' -> True
+    let c = 
+      match s.[0] with
+        [ '<' -> let k = String.index_from s 0 '>' in
```

Comment:
My point is, the name of the function is `start_with_vowel`. Is `<` a vowel ? Clearly no. Which means that if you want to call this function with some string that has html tags, then you must clean your string before calling this function.

Don't you think this is the right approach ?